KeepingYouAwake/KYAAppController/KYAAppController.m

```diff
@@ -320,6 +325,18 @@ - (void)handleActivateActionForEvent:(KYAEvent *)event
     }
 }
 
+- (void)externalMonitorStatusChanged:(NSNotification *)notification
+{
+    if ([[NSScreen screens] count] > 1 && [[NSUserDefaults standardUserDefaults] kya_isActivateOnExternalDisplayConnectedEnabled])
```

Comment:
Can you please use the dot syntax here for class properties and (non-boolean) properties?

`NSScreen.screens.count` and `NSUserDefaults.standardUserDefaults`